web_console/dynamic-plugin/content-security-policy.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="content-security-policy_{context}"]
+= Content Security Policy (CSP)
+include::_attributes/common-attributes.adoc[]
+:context: content-security-policy
+
+toc::[]
+
+You can specify Content Security Policy (CSP) directives for your dynamic plugin using the `contentSecurityPolicy` field in the `ConsolePluginSpec`. This field helps mitigate potential security risks by specifying which sources are allowed for fetching content like scripts, styles, images, and fonts. For dynamic plugins that require loading resources from external sources, defining custom CSP rules ensures secure integration into the {product-title} console.
```

Comment:
the monospaced `ConsolePluginSpec` needs a trailing noun, e.g. parameter, field, value, placeholder etc.